setup.py

```diff
@@ -20,7 +20,7 @@
     zip_safe=False,
     install_requires=['pytest>=3.1.0'],
     setup_requires=['setuptools_scm'],
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
     classifiers=[
         'Development Status :: 7 - Inactive',
```

Comment:
@RonnyPfannschmidt I'm curious if this should be
```suggestion
        'Development Status :: 6 - Mature',
```

Reply:
If you take the maintainer role its your choice

Reply:
I think I don't want full maintainership with leading feature development but help with releases and occasional reviews would be fine.